presto-benchto-benchmarks/README.md

```diff
@@ -67,6 +68,8 @@ macros:
 With the scene set up as in the previous section, the benchmark can be run with:
 ```bash
 ./mvnw clean package -pl presto-benchto-benchmarks
-java -Xmx1g -jar presto-benchto-benchmarks/target/presto-benchto-benchmarks-*-executable.jar \
+java -Xmx1g -jar path/to/benchto-driver.jar \
+    --sql presto-benchto-benchmarks/src/main/resources/sql \
+    --benchmarks presto-benchto-benchmarks/src/main/resources/benchmarks \
```

Comment:
Yes, I think it is the case. However, one still has to set up all prerequisites which are very non-trivial (e.g: benchto-service). When we switch to data storage abstraction this tutorial could be step-by-step introduction how to run benchmarks and log results as text file for instance.